src/libs/runtime-utils.js

```diff
@@ -63,13 +64,14 @@ const dataprocCost = (machineType, numInstances) => {
 export const runtimeConfigBaseCost = config => {
   const {
     cloudService, masterMachineType, masterDiskSize, numberOfWorkers, workerMachineType, workerDiskSize, bootDiskSize
-  } = normalizeRuntimeConfig(
-    config)
+  } = normalizeRuntimeConfig(config)
+
+  const isDataproc = cloudService === cloudServices.DATAPROC
```

Comment:
Here the logic for `isDataproc` utilizes the cloudservices but in NewRuntimeModal the logic is `const isDataproc = !sparkMode && !runtimeConfig?.diskSize` are these supposed to be different?

Reply:
Yea, they are intended to be `const`s local to the functions they are defined in with different meanings.